open-api.yaml

```diff
@@ -960,9 +960,9 @@ paths:
                 Primary Key Already Present:
                   value:
                     message: A primary key is already present. It's impossible to update it
```

Comment:
Missing dot `.` at the end of the sentence. Also i was thinking of the following wording:

`Primary Key's can not be updated when one is already defined in the index.`

Additionaly:
`Index should be re-created with correct primary key`

Reply:
The OAS is not yet up to date, the most updated messages are in the specification file 🤓